core/controllers/dashboard_test.py

```diff
@@ -463,6 +463,92 @@ def test_no_explorations(self):
         self.assertEqual(response['explorations_list'], [])
         self.logout()
 
+    def test_view_dashboard_no_explorations(self):
+        self.login(self.OWNER_EMAIL)
+        response = self.get_json(feconf.DASHBOARD_DATA_URL)
+        self.assertEqual(response['explorations_list'],[])
+
+        # Testing that creator only visit dashboard without any exploration created
+        response = self.get_json(feconf.DASHBOARD_DATA_URL)
+        self.assertEqual(len(response['explorations_list']),0)
+        
+        # Now viewing the dashboard without creating the exploration
+        response = self.get_json(feconf.DASHBOARD_DATA_URL)
+        self.assertEqual(response['explorations_list'],[])
```

Comment:
These 2 lines are being repeated in each test, there's no benefit of doing this. Please remove the repetitions from each test.